tests/run/fused_cpdef.pyx

```diff
@@ -1,24 +1,39 @@
 cimport cython
+import sys, io
 
 cy = __import__("cython")
 
-cpdef func1(self, cython.integral x):
-    print "%s," % (self,),
+cpdef func1(self, cython.integral x, printto=None):
+    if printto is None:
+        printto = sys.stdout
+    print >> printto, "%s," % (self,),
```

Comment:
Since you're adding new code here, please use something Py3 compatible instead of this syntactic wart.

Reply:
Apologies. I wanted to maintain the language version that was already in the file.